src/ipc/StoreMap.h

```diff
@@ -259,9 +259,14 @@ class StoreMap
     /// undoes partial update, unlocks, and cleans up
     void abortUpdating(Update &update);
 
-    /// only works on locked entries; returns nil unless the slice is readable
+    /// the caller must hold a lock on the entry
+    /// \returns nullptr unless the slice is readable
     const Anchor *peekAtReader(const sfileno fileno) const;
 
+    /// the caller must hold a lock on the entry
```

Comment:
I have not checked, but I suspect that placing the `caller must hold a lock on the entry` line on top will make _that_ phrase as the brief method description in Doxygen. That would be unfortunate because the line does not summarize what the method is about. If I am right, consider moving that line down, after the `\returns` line _if_ doing so would solve the problem -- I do not know whether Doxygen is smart enough to use the `\returns` line as the brief method description.

Reply:
No, reordering these lines did not do the trick. I experimented also with different related Doxygen options but could not get 'returns' line in the brief description. The only option I know to overcome this - revert back to a single line description without 'returns' tag.

Reply:
> The only option I know to overcome this - revert back to a single line description without 'returns' tag.

OK, let's leave the descriptions as they are now. Further improvements are probably not worth the time spent fighting Doxygen and reviewers.